tests/kw_include_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+include "./tests/utils"
+
+function suite
+{
+  suite_addTest "include_Test"
+  suite_addTest "include_twice_Test"
+  suite_addTest "include_wrong_path_Test"
+}
+
+function include_Test()
+{
+  ID=1
+  include ./src/kwio.sh
+  expected_result="1"
+  output=$KWIO_IMPORTED
+  assertEquals "($ID)" "$expected_result" "$output"
+}
+
+function include_twice_Test()
+{
+  ID=2
+  include ./src/kwlib.sh
+  include ./src/kwlib.sh
```

Comment:
Just for curiosity, what happens if I include the include?

Reply:
it will source itself again, but if you try it again than it wont be sourced twice(or trice, actually). We can declare a INCLUDE_IMPORTED global variable as well to keep it from sourcing it the first time but I think it is unnecessary. Nobody should even consider including include and this won't protect it from being sourced twice with ` . kw_include etc`. If we are worried about this then we should use an include guard like the original idea for this issue.

Reply:
Don't worry about that... I just asked for curiosity, maybe explain this on the documentation page.